users/README.md

```diff
@@ -101,6 +107,12 @@
     "restricted": false,
     "super_user": false,
     "in_discord" : true,
+  },
+    "archivedDetails": {
+    "updated_by": "July 10, 2023 at 8:51:30 AM UTC+",
```

Comment:
`updated_by` gives a sense who updated it . but the value is a timestamp hence I recommended using updatedAt